### PR TITLE
Add AutomationProperties page

### LINKED ISSUE #199: Add Accessibility sample page
**Is this a totally new sample, an addition to an existing sample page, or a modification of an existing sample page?**
- [x] New sample
- [ ] Addition to existing sample
- [ ] Modification to existing sample

**If this is an addition/modification to an existing sample page, which one?**
N.A.

**Describe the sample**

Since we have samples for various controls and also non controls e.g. RevealStyle which is not a "real" control, one might expect to be able to use the XCG to find out about ways to write accessible UIs. However in XCG there are no samples regarding accessibility.

**Is your sample request related to a problem? Please describe it**

Creating accessible Apps is not easy, and there are a few samples regarding this. However since this is one of the easiest (and IMO one of the best) resources for WinUI components, having no samples that include accessibility features could prevent one from thinking about accessibility. Having samples for the most common accessibility propererties such as AutomationProperties.Name or AutomationProperties.AccessibilityView would make creating accessible components easier.

**Additional context**

This is something also requested by the community (well sort of):
https://github.com/microsoft/microsoft-ui-xaml/issues/1164

### PR DESCRIPTION
**Pull request by chingucoding**
## Description

This PR will add a page containing samples around AutomationProperties, which are useful (and needed) to write accessible XAML.

## Motivation and Context

Closes #199 
## How Has This Been Tested?

Tested manually by starting application.
## Screenshots (if appropriate):
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by chingucoding** [comment]
> The only thing we may want to change is the category where AutomationProperties show up. Right now this page is in Basic Input, which doesn't seem quite right. Maybe Status and Info or Styles would make a better home?

This was something I though about too. My though was that accessible UI should be a basic duty for developers and nothing that should be seen as "special" or optional. But since it is not an input, I think "Status and Info" or "Styles" are better. I would think, "Status and Info" might be the best choice. I can update the PR with the best fitting category if you want.

On a different thought, could we even create a new category "Accessibility" for more sample pages to come?

**Comment by YuliKl** [comment]
> On a different thought, could we even create a new category "Accessibility" for more sample pages to come?

Yes, that's also a good option that would further elevate Accessibility and hopefully draw more attention to the topic. Having only one page in a category isn't great, but maybe that will motivate us to add more pages 😊

**Comment by chingucoding** [comment]
For now I move it into the "Status and Info" for now. Having a category with only one element in it feels a bit weird, but if you think we should do this with this PR I would be happy to do that :)

File: XamlControlsGallery/DataModel/ControlInfoData.json [hunk 1]
```diff
@@ -993,6 +993,29 @@
       "ImagePath": "",
       "Description": "",
       "Items": [
+        {
+          "UniqueId": "AutomationProperties",
+          "Title": "AutomationProperties",
+          "Subtitle": "Attributes that help with writing accessible XAML.",
+          "ImagePath": "ms-appx:///Assets/Button.png",
```
Comment (stmoy) [inline XamlControlsGallery/DataModel/ControlInfoData.json]: Looking forward: we should probably get a more appropriate icon here.

**Pull request merged**